bundlewrap/utils/ui.py

```diff
@@ -96,7 +96,7 @@ def term_width():
 def page_lines(lines):
     lines = list(lines)
     line_width = max([len(line) for line in lines])
-    if line_width > term_width():
+    if TTY and line_width > term_width():
```

Comment:
Ideally, this would use the raw line width, not the one including escape sequences. Doesn't break things but it's a little confusing.